src/test/java/array/Exercise05Test.java

```diff
@@ -0,0 +1,19 @@
+package array;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class Exercise05Test {
+    @Test
+    void shouldFindMissingNumber_OK() {
+        final Exercise05 ex = new Exercise05();
+
+        assertEquals(2, ex.findMissingNumber(new int[]{0, 1, 3}));
```

Comment:
Regarding to the exercise: A whole number of n elements values from 1 to n. Then 0 should not be in the arr.